src/WebJobs.Script.WebHost/Diagnostics/EtwEventGenerator.cs

```diff
@@ -17,25 +17,25 @@ public EtwEventGenerator()
             _azureMonitorEventSource = AzureMonitorDiagnosticLogsEventSource.Instance;
         }
 
-        public void LogFunctionTraceEvent(LogLevel level, string subscriptionId, string appName, string functionName, string eventName, string source, string details, string summary, string exceptionType, string exceptionMessage, string functionInvocationId, string hostInstanceId, string activityId, string runtimeSiteName)
+        public void LogFunctionTraceEvent(LogLevel level, string subscriptionId, string appName, string functionName, string eventName, string source, string details, string summary, string exceptionType, string exceptionMessage, string functionInvocationId, string hostInstanceId, string activityId, string runtimeSiteName, string slotName)
```

Comment:
I know the issue only states FunctionsLogs, but should we be doing this for FunctionsMetrics as well while we're in here? @jeffhollan?

Reply:
Good question and I assume so. I created this on behalf of @fabiocav but I think it makes sense

Reply:
It would help to have that in the metrics as well.